tools/amp_ner/ner_to_csv.xml

```diff
@@ -0,0 +1,22 @@
+<tool id="ner_to_csv" name="AMP Named Entity JSON to CSV Converter" version="1.1.0">
+  <description>Convert NER output to CSV</description>
+  <requirements>
+  </requirements>
```

Comment:
We should add python as one requirement, see hmgm_ner.xml for example.

In general (not the case here) if there're other library packages (those not included in standard python) we should put in the requirements with its versions too.